code/game/machinery/station_holomap.dm

```diff
@@ -185,6 +192,22 @@
 
 // TODO: Make these constructable.
 
+/obj/machinery/station_map/mobile
+	idle_power_usage = 0
+	active_power_usage = 0
```

Comment:
`POWER_USE_OFF` macro?

Reply:
That's for `use_power`, does not seem like just setting `user_power` to `POWER_USE_OFF` will stop it from using power.